PullRequestCreated/ReadMe.md

```diff
@@ -25,6 +25,16 @@ Token to access the Jira API.
 
```

Comment:
Please update the intro to document the functionality that this action does. Like updating issue title and issue description